Wordle.UI/ConsoleUI.cs

```diff
@@ -0,0 +1,124 @@
+using Wordle.Domain;
+namespace Wordle.UI;
+
+public class ConsoleUI : IWordleUI
+{
+    Game game = new Game();
+    GuessValidator guessValidator = new GuessValidator();
```

Comment:
The Game should not exist within the UI.  If anything, all three of these should be in "main", and there could be some kind of injecting going on.  With this design, every UI would require its own game.